src/Components/SceneBreadcrumb/Internal/SceneDropdown.tsx

```diff
@@ -5,7 +5,10 @@ import { ADT3DScenePageContext } from '../../../Pages/ADT3DScenePage/ADT3DSceneP
 import { dropdownStyles } from './SceneDropdown.styles';
 import { SceneDropdownProps } from './SceneDropdown.types';
 
-const SceneDropdown: React.FC<SceneDropdownProps> = ({ sceneId }) => {
+const SceneDropdown: React.FC<SceneDropdownProps> = ({
+    sceneId,
+    onCloseBehaviorsModal
```

Comment:
I like the pattern, but would rather use a name like `onSelectionChange` or `onChange`.